lisp/ess-r-mode.el

```diff
@@ -2267,7 +2269,36 @@ state.")
         ess-help-sec-keys-alist ess-help-r-sec-keys-alist ; TODO: Still necessary?
         inferior-ess-help-command inferior-ess-r-help-command
         ess-get-help-topics-function #'ess-s-get-help-topics-function
-        inferior-ess-help-filetype nil))
+        inferior-ess-help-filetype nil)
+  (ess-r-help-add-links))
+
+(define-button-type 'ess-r-help-link
+  'follow-link t
+  'action (lambda (_) (ess-r-help-button-action)))
+
+(defun ess-r-help-button-action ()
+  "Display help for button at point."
+  (let ((text (get-text-property (point) 'ess-r-help-link-text)))
+    (ess-display-help-on-object text)))
+
+(defun ess-r-help-add-links ()
+  "Add links to the help buffer."
+  (save-excursion
+    ;; Search for fancy quotes only. If users have
+    ;; options(useFancyQuotes) set to something other than TRUE this
+    ;; probably won't work. If it's FALSE, R outputs ascii ', but
+    ;; searching through the whole buffer takes too long.
+    (while (re-search-forward "‘[^[:space:]]+?’" nil t)
+      (let ((inhibit-read-only t)
+            (beg (match-beginning 0))
+            (end (match-end 0))
+            ;; Remove surrounding quotes so the text property is the
+            ;; name of the link:
+            (text (string-trim (match-string 0) "[\"‘]*" "[\"’]*")))
```

Comment:
Line 2297 doesn't compile for me.   Perhaps
```
(text (replace-regexp-in-string "[\"']" "" (match-string 0))
```

Reply:
Thanks, fixed. string-trim didn't gain arguments until emacs 26.1.